resources/views/dashboard.blade.php

```diff
@@ -36,6 +53,16 @@
                       <?php $balanceStyle = array(['class'=>'', 'operator' => '']); ?>
                       @endif
                           <tr>
+                              <td>
+                                  @foreach ($allSharedExpenses as $sharedExpense)
```

Comment:
I fixed this now @danielcapacio 

![image](https://cloud.githubusercontent.com/assets/15661248/24998414/e4ad55f8-1fee-11e7-906f-6b1f1648e1cd.png)